pkg/ec2helper/ec2helper.go

```diff
@@ -975,15 +1008,30 @@ func getRunInstanceInput(simpleConfig *config.SimpleInfo, detailedConfig *config
 			}
 		}
 
-		// Set auto-termination timer
-		if IsLinux(*detailedConfig.Image.PlatformDetails) && simpleConfig.AutoTerminationTimerMinutes > 0 {
+		// Set auto-termination
+		setAutoTermination = IsLinux(*detailedConfig.Image.PlatformDetails) && simpleConfig.AutoTerminationTimerMinutes > 0
+		if setAutoTermination {
 			input.InstanceInitiatedShutdownBehavior = aws.String("terminate")
-			userData := []byte(fmt.Sprintf("#!/bin/bash\necho \"sudo poweroff\" | at now + %d minutes",
-				simpleConfig.AutoTerminationTimerMinutes))
-			input.UserData = aws.String(base64.StdEncoding.EncodeToString(userData))
 		}
 	}
 
+	if simpleConfig.UserDataFilePath != "" {
+		userDataRaw, _ := ioutil.ReadFile(simpleConfig.UserDataFilePath)
```

Comment:
does os.Stat handle all kinds of errors involving file read? Double checking if the err should be checked here.

Reply:
another refactor that's on my list.. but basically I'm skipping the `error` here because the validation checks on the filepath would have exited on **any** error

This should be consolidated in one place